### ISSUE TITLE
The Narrate menu becomes unresponsive if it's opened on a second article while another one is still active

### ISSUE THREAD

**Issue by Softvision-PatriciuPop** [mozilla/firefox-narrate-experiment#16]
[Affected versions]:
- Firefox Unbranded Beta Version 66.0b9, Build ID 20190218131312

[Affected Platforms]:
- All Windows
- All Linux

[Prerequisites]:
- The "xpinstall.signatures.dev-root" boolean pref was created with the "true" value.
- The firefox-listen-study@shield.mozilla.org-0.3-signed-testing.xpi is installed.
- Enrolled in the feature-passive or feature-active branch.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. Navigate to a Wikipedia article and click the "Toggle Reader View" button.
3. Navigate to a Wikipedia article in a new tab and click the "Toggle Reader View" button.
4. Interact with the Narrate control menu and observe the behavior.

[Expected result]:
- The second article doesn't automatically play.

[Actual result]:
- The control menu shows the text as being narrated and doesn't responds to any interactions.

[Notes]:
- If the first narrate playback is stopped, the frozen one remains unresponsive until the tab is refreshed.
- The behavior is different on Mac OS as described in bug #15 .
- I have attached a screen recording of the issue:
![fxnarrate_multi](https://user-images.githubusercontent.com/40393891/53179892-4446cf00-35fd-11e9-9f93-cf7d7d07093c.gif)

**Comment by awallin** [comment]
Closing as this is an existing issue with the Narrate feature unrelated to this experiment/add-on.

**Comment by Softvision-PatriciuPop** [comment]
Indeed, it's more of an edge case on the feature. However, it is more easy to encounter it on the add-on. This is caused by the auto play feature of the add-on causing it to appear every time a second article is opened in Reader View mode, not only in the narration breaks as on the feature (existing issue).
From the QA's point of view this could have a negative impact on the users, but it's your decision after all if you want to fix this on the add-on or not.

**Comment by awallin** [comment]
I completely understand the concern. This particular feature (Narrate) has a number of issues that have remained unaddressed for years (this one being a prime example) because there's no visibility into the numbers of users and/or desire of people to listen.

My reasoning for pushing this experiment without fixing this bug is without measurable user value (e.g. a lot of people clicking "Listen") we can't justify/demonstrate the value in making these types of fixes.

Basically, if we get a lot of users encountering the problem documented in this issue it's a strong signal there's value in making the fix. Without that there are no resources to make the fix.
